cdc/generation_id.hh

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
```

Comment:
I think this is the wrong license. None of the CDC code is taken from Apache Cassandra... Compare the license blurbs in alternator/, where also nothing was copied from Cassandra.

Reply:
Fixed